core/amber/src/main/scala/edu/uci/ics/texera/workflow/operators/typecasting/TypeCastingOpDesc.java

```diff
@@ -60,18 +60,26 @@ public Schema getOutputSchema(Schema[] schemas) {
         List<String> attributeNames = schemas[0].getAttributeNames();
         List<AttributeType> attributeTypes = attributes.stream().map(attr -> attr.getType()).collect(toList());
         Schema.Builder builder = Schema.newBuilder();
+        // this loop check whether the current attribute in the array is the attribute for casting,
+        // if it is, change it to result type
+        // if it's not, remain the same type
+        // we need this loop to keep the order the same as the original
         for (int i=0;i<attributes.size();i++) {
             if (attributeNames.get(i).equals(attribute)) {
                 if (this.resultType != null){
                     switch (this.resultType) {
                         case STRING:
                             builder.add(this.attribute, AttributeType.STRING);
+                            break;
                         case BOOLEAN:
                             builder.add(this.attribute, AttributeType.BOOLEAN);
+                            break;
                         case DOUBLE:
                             builder.add(this.attribute, AttributeType.DOUBLE);
+                            break;
                         case INTEGER:
                             builder.add(this.attribute, AttributeType.INTEGER);
+                            break;
                     }
```

Comment:
For any branching code, including `if-branch`, `switch-branch`, please make sure all possible cases are handled. Here you are missing `LONG`, `TIMESTAMP` and `ANY` types.

Reply:
Since it didn't have `LONG` and `TIMESTAMP` when I make typecasting operator, I didn't make it previously.
I wonder should I include the change in this PR or a new PR related to `type utilities`?

Reply:
you can leave that to later PR, but here please handle any case other than these four (by default, raise exception)

Reply:
please add some error messages in the exception